internal/operators/cnv/cnv_operator_test.go

```diff
@@ -0,0 +1,146 @@
+package cnv_test
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/host/hostutil"
+	"github.com/openshift/assisted-service/internal/operators/api"
+	"github.com/openshift/assisted-service/internal/operators/cnv"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+const supportedGPUsEnv = "MYAPP_CNV_SUPPORTED_GPUS"
+
+var _ = Describe("CNV operator", func() {
+	var (
+		log      = logrus.New()
+		operator api.Operator
+	)
+
+	BeforeEach(func() {
+		_ = os.Unsetenv(supportedGPUsEnv)
```

Comment:
We shouldn't set and unset environment variables in UT, since it could affect other tests that run in parallel. The config should be provided to the CTOR of the class. It would be much easier to be tested.